unlock-js/src/v13/initializeTemplate.js

```diff
@@ -0,0 +1,30 @@
+import { ZERO } from '../constants'
+
+/**
+ * Initialize a lock (will be called on the template only)
+ * @param {PropTypes.contract} contract
+ */
+export default async function({ templateAddress }, callback) {
+  const signer = this.provider.getSigner()
+  const _owner = await signer.getAddress()
```

Comment:
Do these need to be underscored? It looks like there are no collisions inside this function definition

Reply:
Probably not, I'll change them. It's just that these are the argument names on the smart contract and I used the same ;)